spec/support/vcr_cassettes/sm_client/messages/creates/cannot_send_draft_as_reply.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Actually, wait... nm, i guess this tests the call to history. This is OK. ignore previous comment.